script/CheckSecurityConfigs.s.sol

```diff
@@ -0,0 +1,185 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { console2 } from "forge-std/console2.sol";
+import { Script } from "forge-std/Script.sol";
+import { stdStorage } from "forge-std/StdStorage.sol";
+
+/**
+ * @title CheckSecuityConfigs
+ * @notice A script to check security configurations of an OP Chain,
+           such as upgrade key holder, challenger and guadian designations.
+ *         The usage is as follows:
+ *         $ forge script CheckSecuityConfigs \
+ *             --rpc-url $ETH_RPC_URL
+ */
+
+contract CheckSecuityConfigs is Script {
+    struct ProtocolAddresses {
```

Comment:
For both it's a bit tricky since the contracts don't live in this repo, and I don't think we want to install the monorepo as a dependency. 

I'm ok with leaving this as-is for now, and opening an issue to think about ways to make this more robust in a future PR